src/main/java/com/puppycrawl/tools/checkstyle/checks/DescendantTokenCheck.java

```diff
@@ -272,17 +571,46 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class Test {
+ *   public void foo() { // OK
+ *       System.out.println("Hello");
+ *   }
+ *   public void boo() {
+ *       ; // Violation
+ *   }
+ * }
+ * </pre>
+ * <p>
  * To configure the check to produce a violation on a class with too many fields:
```

Comment:
Please remove this example completely, or combine it with `interface with too many fields`, config and idea are the same.